lib/fastlane/plugin/wpmreleasetoolkit/actions/common/prototype_build_details_comment_action.rb

```diff
@@ -0,0 +1,130 @@
+module Fastlane
+  module Actions
+    class PrototypeBuildDetailsCommentAction < Action
+      def self.run(params)
+        # Get Input Parameters
+        appcenter_org_name = params[:appcenter_org_name]
+        appcenter_app_name = params[:appcenter_app_name]
+        appcenter_release_id = params[:appcenter_release_id]
+        commit = params[:commit] || other_action.last_git_commit[:abbreviated_commit_hash]
+
+        metadata = params[:metadata] # e.g. {'Build Config':… , 'Version': …, 'Short Version': …}
+        direct_link = params[:download_url]
+        unless direct_link.nil?
+          metadata['Direct Link'] = "<a href='#{direct_link}'><tt>#{File.basename(direct_link)}</tt></a>"
+        end
+
+        # Build the comment parts
+        install_url = "https://install.appcenter.ms/orgs/#{appcenter_org_name}/apps/#{appcenter_app_name}/releases/#{appcenter_release_id}"
+        qr_code_url = "https://chart.googleapis.com/chart?chs=500x500&cht=qr&chl=#{CGI.escape(install_url)}&choe=UTF-8"
+        metadata_rows = metadata.compact.map do |key, value|
+          "<tr><td><b>#{key}</b></td><td>#{value}</td></tr>"
+        end
+
+        intro = "📲 You can test the changes from this Pull Request in <strong>#{appcenter_app_name}</strong> by scanning the QR code below to install the corresponding build via App Center."
```

Comment:
I noticed the usage of `<b>` in the table rows and `<strong>` in this intro text.

That sent me into a mini-rabbit-hole to understand the difference between the two. I found the [`<strong>` documentation from Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong) useful:

> The `<strong>` element is for content that is of greater importance, while the `<b>` element is used to draw attention to text without indicating that it's more important.
>
> It may help to realize that both are valid and semantic elements in HTML and that it's a coincidence that they both have the same default styling (boldface) in most browsers (although some older browsers actually underline `<strong>`). 

With that in mind, I'd suggest making this `<b>`. At the same time, I don't think we need to sweat the HTML semantics for the content of a comment going in a GitHub PR...

Reply:
Addressed in 4204a4532477907f124a32b01f39cccddcfdece9